src/components/play-pause/play-pause.tsx

```diff
@@ -57,12 +59,14 @@ class PlayPause extends Component<any, any> {
    * @memberof PlayPause
    */
   componentDidMount(): void {
-    const {eventManager, player} = this.props;
-    eventManager.listenOnce(player, player.Event.UI.USER_CLICKED_PLAY, () => {
-      eventManager.listenOnce(player, player.Event.Core.FIRST_PLAY, () => {
-        this._playPauseButtonRef?.focus();
+    const {eventManager, player, guiClientRect} = this.props;
```

Comment:
please double check that this works in playlist playback, since componentDidMount, theoretically, is called once per component

since this code already existed, i assume it works (and the component is destroyed and recreated for every new entry loaded), but please make sure

Reply:
originally with voiceOver and playlists, after loading a new entry, the focus shifted back to the whole window.
my fix is not affecting this behavior, meaning the focus still shifts back to the window after loading a new entry.
but I think this is another behavior that is not related to the play/pause functionality and this issue specifically.
let me know if you agree or not.